airflow/docker.go

```diff
@@ -27,6 +27,8 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+var errComposeProjectRunning = errors.New("project is up and running")
```

Comment:
Maybe it would be best to put all the error message in a `airflow/errors.go` file.

So it the package grows in the futur, we know where to look for error messages

Reply:
moved to a separate file